components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/openidconnect/RequestObjectValidatorImplTest.java

```diff
@@ -222,4 +284,55 @@ private void mockIdentityProviderManager() throws Exception {
         when(IdentityProviderManager.getInstance()).thenReturn(identityProviderManager);
         when(identityProviderManager.getResidentIdP(anyString())).thenReturn(idp);
     }
+
+    @DataProvider(name = "nbfExpDataProvider")
+    public Object[][] getNbfExpClaims() {
+        long currentTimeMillis = System.currentTimeMillis();
+        Date timeInLastHour = new Date(currentTimeMillis - TimeUnit.MINUTES.toMillis(30));
+        Date timeInNextHour = new Date(currentTimeMillis + TimeUnit.MINUTES.toMillis(30));
+
+        return new Object[][] {
+                {new Date(currentTimeMillis - TimeUnit.MINUTES.toMillis(70)), timeInNextHour, false,
+                        "Request Object nbf claim is too old."},
+                {timeInLastHour, timeInNextHour, true, null},
+                {new Date(currentTimeMillis + TimeUnit.MINUTES.toMillis(10)), timeInNextHour, false,
+                        "Request Object is not valid yet."},
+                {null, timeInNextHour, false, "Request Object does not contain Not Before Time."},
+                { timeInLastHour, new Date(timeInLastHour.getTime() + TimeUnit.MINUTES.toMillis(50)), true,
```

Comment:
make formatting consistent
```suggestion
                {timeInLastHour, new Date(timeInLastHour.getTime() + TimeUnit.MINUTES.toMillis(50)), true,
```